drivers/cas_access/app/models/cas_access/opportunity.rb

```diff
@@ -9,5 +9,12 @@ class Opportunity < CasBase
     self.table_name = :opportunities
     belongs_to :voucher, optional: true
     has_many :programs, through: :voucher
+    has_many :opportunity_contacts
+
+    has_one :status_match, -> { where arel_table[:active].eq(true).or(arel_table[:closed].eq(true).and(arel_table[:closed_reason].eq('success'))) }, class_name: 'ClientOpportunityMatch'
+    has_many :closed_matches, -> do
+      where(closed: true).
+        order(updated_at: :desc)
+    end, class_name: 'ClientOpportunityMatch'
```

Comment:
Is this in use?  I think we went with the scope on client opportunity match.

Reply:
you're correct. I missed taking this out after the change.